documentation/release_notes.markdown

```diff
@@ -8,42 +8,75 @@ canonical: "/puppetdb/latest/release_notes.html"
 [kahadb_corruption]: ./trouble_kahadb_corruption.html
 [pg_trgm]: http://www.postgresql.org/docs/current/static/pgtrgm.html
 [upgrading]: ./api/query/v4/preparing-for-3.0.html
+[puppetdb-module]: https://forge.puppetlabs.com/puppetlabs/puppetdb
 
 3.0.0
 -----
 
-### Upgrading
-
-* For the best-possible performance and scaling capacity, we recommend
- PostgreSQL version 9.4 or newer with the [`pg_trgm`][pg_trgm]
- extension enabled, as explained [here][configure_postgres], and we
- have officially retired versions earlier than 9.4. HSQLDB is
- only recommended for local development because it has a number of
- scaling and operational issues.
+PuppetDB 3.0.0 is a major release which introduces some major new features,
+with some breaking changes.
 
-* Make sure that all of your PuppetDB instances are shut down, and
- only upgrade one at a time.
+### Contributors
 
-* Make sure to upgrade your puppetdb-terminus package (on the host
- where your Puppet Master lives), and restart your master service.
+Andrew Roetker, Andrii Nikitiuk, Chris Price, Eric Sorenson, John Duarte,
+Jarie Tappa, Ken Barber, Brian LaMattery, Lars Windolf, Mathieu Parent,
+Matthaus Owens, Nick Fagerlund, Rob Braden, Rob Browning, Russell Mull,
+Ryan Senior, Scott Garman, Wayne Warren, Wyatt Alt, Jean Bond and
+Darin Perusich.
 
-* Ensure during a package upgrade that you analyze any changed
-  configuration files. For Debian you will receive warnings when
-  upgrading interactively about these files, and for RedHat based
-  distributions you will find that the RPM drops .rpmnew files that
-  you should diff and ensure that any new content is merged into your
-  existing configuration.
+### Upgrading
 
-* There are some migrations provided in this release, that may mean
-  you have to wait while these run after upgrading. Before upgrading
-  you should also ensure you have as much free disk space as you have
-  data, as migrations often require temporary duplication of table
-  data to succeed.
+* This version of PuppetDB adopts the AIO layout to be consistent with other
+  Puppet Labs layouts, such as Puppet Server.
+
+    This means the following paths have changed:
+
+    * Primary configuration: /etc/puppetdb is now /etc/puppetlabs/puppetdb
+    * Logs: /var/log/puppetdb is now /var/log/puppetlabs/puppetdb
+    * Binaries: /usr/bin is now /opt/puppetlabs/bin
+    * Data: /var/lib/puppetdb is now /opt/puppetlabs/server/data/puppetdb
+    * Rundir: /var/run/puppetdb is now /var/run/puppetlabs/puppetdb
+
+    During upgrade we have written a script that will migrate your configuration
+    to the new locations. After upgrade you should double check the migrated
+    configuration to ensure everything is correct. Of course we recommend the
+    use of the [PuppetDB Puppet Module][puppetdb-module] version 5, as it will
```

Comment:
I might put this at the first note, hightlight it, maybe put it in a big jumbotron-esque font